waku/factory/external_config.nim

```diff
@@ -300,31 +306,12 @@ hence would have reachability issues.""",
       name: "rln-relay-dynamic"
     .}: bool
 
-    rlnRelayIdKey* {.
-      desc: "Rln relay identity secret key as a Hex string",
-      defaultValue: "",
-      name: "rln-relay-id-key"
-    .}: string
-
-    rlnRelayIdCommitmentKey* {.
-      desc: "Rln relay identity commitment key as a Hex string",
-      defaultValue: "",
-      name: "rln-relay-id-commitment-key"
-    .}: string
-
     rlnRelayTreePath* {.
       desc: "Path to the RLN merkle tree sled db (https://github.com/spacejam/sled)",
       defaultValue: "",
       name: "rln-relay-tree-path"
     .}: string
 
-    rlnRelayBandwidthThreshold* {.
```

Comment:
the one deleted were actually not used

Reply:
I would suggest instead to archive the rln-interep-contract repo cc @jm-clius @kaiserd